include/tim/vx/operation.h

```diff
@@ -49,6 +49,7 @@ class Operation {
   std::unique_ptr<OpImpl>& impl();
   const std::unique_ptr<OpImpl>& impl() const;
   virtual const std::vector<std::shared_ptr<Tensor>> ConstantInputsTensor() const;
+  virtual void HandleAfterBindInput(const std::shared_ptr<Tensor>& tensor, int32_t input_idx);
```

Comment:
rename it as OnBindInputPostProc and make sure it is a private method not public.
And addd comments for it.

Reply:
Done